addons/website_sale/controllers/main.py

```diff
@@ -1095,13 +1095,21 @@ def shop_payment_confirmation(self, **post):
         sale_order_id = request.session.get('sale_last_order_id')
         if sale_order_id:
             order = request.env['sale.order'].sudo().browse(sale_order_id)
-            return request.render("website_sale.confirmation", {
-                'order': order,
-                'order_tracking_info': self.order_2_return_dict(order),
-            })
+            values = self._prepare_confirmation_values(order)
+            return request.render("website_sale.confirmation", values)
         else:
             return request.redirect('/shop')
 
+    def _prepare_confirmation_values(self, order):
```

Comment:
Seems a method called with the same name now exist on sale order model, but is not involved in frontend. I would rename to avoid conflicts ^^